project-koku/masu
# Consolidate common downloader logic

Closes #236 

This consolidates some of the report download logic into the top-level `ReportDownloader` class.  This was important to do because there is now a single place populating the report dictionary structures that are passed on to the processor once downloaded.  It also enabled me to lift the lint suppression for `OCPReportDownloader`.

*Testing*
1.  Add an account for AWS, AWS-local, and OCP in koku
2.  Ensure download/processing is successful in masu

*Test Results*
[masu_download_refactor_ut.txt](https://github.com/project-koku/masu/files/2484884/masu_download_refactor_ut.txt)
